plugins/CMakeLists.txt

```diff
@@ -92,4 +92,9 @@ add_subdirectory(obs-transitions)
 add_subdirectory(rtmp-services)
 add_subdirectory(text-freetype2)
 add_subdirectory(aja)
-add_subdirectory(obs-websocket)
+
+if(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/obs-websocket/CMakeLists.txt)
```

Comment:
We should have a `check_obs_websocket()` function for this just like `check_obs_browser()` since they are now our only two plugin submodules

Reply:
This function was made because this plugin is not available on FreeBSD and so we needed to the check for Windows, macOS and Linux.

Reply:
I think then at least moving these lines towards the top of the file would help flow. Right now it's logic followed by includes then followed again by more logic; just doesn't feel right.

Reply:
Moved the code just after the Windows/macOS/Linux/FreeBSD ifs.